src/lib/layout/index.spec.ts

```diff
@@ -517,21 +519,15 @@ describe('nexusModules', () => {
       'app.ts': `import { schema } from 'nexus'`,
       'graphql.ts': `import { schema } from 'nexus'`,
     })
-    const result = await ctx.createLayoutThrow({ entrypointPath: './app.ts' })
-    expect({
-      app: result.app,
-      nexusModules: result.nexusModules,
-    }).toMatchInlineSnapshot(`
-          Object {
-            "app": Object {
-              "exists": true,
-              "path": "__DYNAMIC__/app.ts",
-            },
-            "nexusModules": Array [
-              "__DYNAMIC__/graphql.ts",
-            ],
-          }
-      `)
+    const result = await ctx.createLayout2({ entrypointPath: './app.ts' }).then(rightOrThrow)
+    // result.nexusModules
+    expect(result).toMatchObject({
+      app: {
+        exists: true,
+        path: expect.stringContaining(ctx.fs.path('app.ts')),
+      },
+      nexusModules: [ctx.fs.path('graphql.ts')],
+    })
```

Comment:
Almost like a snapshot but it has the advantage of letting us pass a dynamic matcher for the path strings. A downside here is that we don't benefit from the snapshot update workflow anymore.